dreammall-earth/dreammall.earth
# refactor(frontend): remove build server script

Motivation
----------
Side quest of #1994.

I tried to rebuild the entire frontend with vike from scratch, following
the current documentation. One thing that I noticed is that they have no
build step for the server code. They simply run `tsx` in production.

This is a good option for us, too, because we can get rid of a lot of
code and complexity.

How to test
-----------
1. Deploy this to the kubernetes cluster
2. Deploy this on our vServer
3. Check if both frontend and presenter are running fine
